napari/components/_dims/__init__.py

```diff
@@ -1 +1 @@
-from .model import Dims
+from .dims import Dims
```

Comment:
Why rename? Isn't `model` a lot cleaner/clearer of a name???

Reply:
I go back and forth on this. I think I'm ok with `dims`, but we should just be consistent, and if we do this remove all usage of the word `model` and replace all usage of `view` with `_qt`

Reply:
I think `modal` and `view` are better options. Love to see those.

Reply:
ok I will switch back to `model` and `view` unless @royerloic strongly objects